unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarLOD/AvatarsLODController.cs

```diff
@@ -134,6 +144,16 @@ internal void UpdateAllLODs(int maxAvatars = DataStore.DataStore_AvatarsLOD.DEFA
                 if (impostorCount < maxImpostors)
                 {
                     lodController.SetImpostor();
+
+                    var distanceToPlayer = Mathf.Sqrt(sqrtDistance);
```

Comment:
If we need the real Distance (I would try to use the Squared distance although it doesn't behave linearly like the distance) we could just get it in `ComposeLODControllersSortedByDistance`.

Reply:
Ended up correcting the tint min and max distances and using the squared distance, check it out

Reply:
Reverted it, check https://github.com/decentraland/unity-renderer/pull/988#discussion_r699666914